lib/public/components/common/popover/overflowBalloon.js

```diff
@@ -53,16 +53,33 @@ export const overflowBalloon = (content, options = null) => {
              */
             onTriggerNodeChange: function (previousTriggerNode, newTriggerNode) {
                 if (previousTriggerNode) {
-                    const node = stretch ? previousTriggerNode.parentNode : previousTriggerNode;
-
-                    node.removeEventListener('mouseover', this.showPopover);
-                    node.removeEventListener('mouseleave', this.hidePopover);
+                    previousTriggerNode.removeEventListener('mouseover', this.prevousNodeMouseoverCallback);
+                    previousTriggerNode.removeEventListener('mouseleave', this.hidePopover);
                 }
 
                 if (newTriggerNode) {
                     const node = stretch ? newTriggerNode.parentNode : newTriggerNode;
+                    const key = newTriggerNode.getAttribute('data-popover-key');
+
+                    const newNodeMouseoverCallback = () => {
+                        this.showPopover();
+                        const selector = `.popover[data-popover-key="${key}"]`;
+                        const contentNode = document.querySelector(selector);
+
+                        contentNode.addEventListener('mouseover', () => {
```

Comment:
Can it be that the `querySelector` will fail because of invalid id? 
Should it be checked that contentNode is not null before adding listeners?